client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -323,11 +323,11 @@ export default function getThankYouPageUrl( {
 		return newBlogReceiptUrl;
 	}
 
-	// disable upsell for tailored signup users
-	const isTailoredSignup = isTailoredSignupFlow( signupFlowName );
+	// disable upsell for given tailored signup users
+	const isUpsellIgnoredTailoredFlow = isUpsellIgnoredTailoredSignupFlow( signupFlowName );
```

Comment:
An unrelated refactor so that the code is explicit that not all tailored flows are ignoring upsells.

Reply:
I can see why this is done but let's revert it back :) 
This change actually inspires me to have a better understanding of what we mean by tailored flows. For the time being, we often refer "flows that implements on Stepper" with it interchangeably, but now I think it's wrong. We have plenty of other flows implemented on Stepper, but those called "tailored flows" are those tailored for specific audience, providing specific onboarding experience that's "tailored" the theme, the placeholder content, etc. for their needs. 

Thus, I found it confusing that we rename this function as "isUpsellIgnoredTailoredFlow", since it implies there is other tailored flows that upsell, while there is actually none.

Does that make sense to you?

Reply:
I actually was not sure whether there were other flows or not. I just thought this name `isTailoredSignupFlow` conveyed an incorrect message that all tailored flows are ignored. But if that is what we want to convey here, we can probably leave that 👍